docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst

```diff
@@ -72,27 +73,52 @@ Run the dihedral fit to fit to the MM simulations:
     fit_dihedral_with_gomc(
         fit_dihedral_atom_types,
         mol2_file,
-        FF_XML_file,
+        forcefield_file,
         temperature_in_unyt_units,
         gomc_binary_directory,
-        log_files_and_removed_points,
-        zeroed_dihedral_atom_types=zeroed_dihedrals,
+        qm_log_file_dict,
+        zero_dihedral_atom_types=zero_dihedral_atom_types,
         qm_engine="gaussian",
-        VDWGeometricSigma=VDWGeometricSigma_bool,
+        combining_rule=combining_rule,
         atom_type_naming_style='general',
         gomc_cpu_cores=1,
-        fit_min_validated_r_squared=0.99,
-        fit_validation_r_squared_rtol=1e-03
+        r_squared_min=0.99,
+        r_squared_rtol=1e-03
     )
 
+The most important output files:
+
+.. code:: ipython3
+
+    # The OPLS dihedral fit constants
+    print('The OPLS dihedral fit constants:\n')
+    os.system("cat opls_dihedral_k_constants_fit_energy.txt")
+
+    # The converted OPLS to periodic dihedral fit constants
+    print('The converted OPLS to periodic dihedral fit constants:\n')
+    os.system("cat periodic_dihedral_k_constants_fit_energy.txt")
+
+    # The converted OPLS to RB-torsions dihedral fit constants
+    print('The converted OPLS to RB-torsions dihedral fit constants:\n')
+    os.system("cat RB_torsion_k_constants_fit_energy.txt")
+
+    # This file contains the raw points used in the fits (QM - MM_less_dihedral(s)_energy being fit)
+    print('This file contains the raw points used in the fits (QM - MM_less_dihedral(s)_energy being fit):\n')
+    os.system("cat all_normalized_energies_in_kcal_per_mol.txt")
+
+
+The plotted dihedral fits:
+   #. opls_all_single_fit_dihedral_k_constants_figure.pdf
+   #. opls_all_summed_dihedrals_k_constants_figure.pdf
+
 
 Fit a dihedral for propionic acid using Gaussian-style output files
 -------------------------------------------------------------------
 The propionic acid CT-CT-C-OH dihedral fit example is provided below. This is an example where the CT-CT-C-OH is fit for both
 CT-CT-C-OH and CT-CT-C-O, by fitting the CT-CT-C-OH and zeroing the CT-CT-C-O, which makes it easier to fit.
 
 Additionally, one can perform the dihedral fit with QM calculations with any QM engine as long as they are in the
-`Gaussian style output files format <https://github.com/GOMC-WSU/MoSDeF-dihedral-fit/tree/main/mosdef_dihedral_fit/utils/files/gaussian_style_output_files>`_
+`Gaussian style output files format <https://github.com/bc118/MoSDeF-dihedral-fit/tree/main/mosdef_dihedral_fit/tests/files/gaussian_style_output_files>`_
```

Comment:
Do you want this to point to your branch of this package and not the parent one at GOMC-WSU?

Reply:
Good Catch.  Changed now